Common/Common.csproj

```diff
@@ -7,6 +7,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore" Version="2.1.1" />
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.1.1" />
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.1.1" />

```

Comment:
what's this?

Reply:
Hmm, i really don't know. I installed NuGet package because of changing the stucture of the projects and some lib's didn't have the packages they needed. Maybe it's cause of that